code/modules/power/smes.dm

```diff
@@ -64,9 +64,10 @@
 					if(term && term.dir == turn(d, 180))
 						terminal = term
 						break dir_loop
-		if(!terminal)
+		/*if(!terminal)

```

Comment:
Are you sure you just want to comment out code? Commenting out code is useful for quick testing and experimentation, but I don't think it's good practice for production code.

Reply:
Yes, this code is called when we first spawn the SMES which doesn't make much sense since we only create the terminal AFTERWARDS.
This is false for original placed SMES which need to actually find their terminal in order to be up and running at the shift start, else engineering would have to set up a lot of stuff

Reply:
The code above which i mentioned was too afraid to remove, works for the original placed SMES as they always find a terminal. Just doesn't make sense setting a SMES to completly broken if it doesn't find a Terminal, when you can only build a terminal when you set the SMES up. Another approach would be to make BROKEN flag disappear after constructing the Terminal.

Reply:
Want me to check any of those mentioned ideas ?

Reply:
Also, you talked about 'commenting out code' you're absolutely right i should've just deleted yeah